db/seeds.rb

```diff
@@ -2572,6 +2572,9 @@ def require_updates?(real_attrs, seed_attrs)
   url: "/jobs_jobs_jobs/conclusion.md"
 )
 
+#create dummy users and projects for the google homepage project
+load './db/seeds/user_seeds.rb'
```

Comment:
Can we change the name of the file to `dummy_projects.rb` please? that's more descriptive as to what this seeds for imo.